internal/dao/pod.go

```diff
@@ -491,3 +499,20 @@ func (p *Pod) isControlled(path string) (string, bool, error) {
 	}
 	return "", false, nil
 }
+
+func getDefaultLogContainer(po v1.Pod) string {
+	defaultContainer := po.GetAnnotations()[defaultLogContainerAnnotation]
```

Comment:
Map access returns v, ok. Just check !ok and return ""

Reply:
done